client/ayon_core/version.py

```diff
@@ -1,3 +1,3 @@
 # -*- coding: utf-8 -*-
 """Package declaring AYON core addon version."""
-__version__ = "0.3.3-dev.1"
+__version__ = "0.3.4-dev.1"
```

Comment:
Please change the core version back.

Reply:
Shouldn't this PR require both an update to maya version and core version since it relies on updated functions to be available in core?

Reply:
PRs should not bump version of core addon.